README.md

```diff
@@ -1 +1,114 @@
 # Apache Pulsar Ruby client
+
```

Comment:
I think it would be really useful to put just a broad checklist of the status of this as far as features, for example, let's call out that batch publishing and batch ack don't work yet.